docs/root/operations/cli.rst

```diff
@@ -232,6 +232,10 @@ following are the command line options that Envoy supports.
   *(optional)* This flag disables Envoy hot restart for builds that have it enabled. By default, hot
   restart is enabled.
 
+.. option:: --listen_for_signals <string>
+
+  *(optional)* A comma-separated list of system signals which Envoy will listen for. For example, `--listen_for_signals SIGTERM,SIGUSR1` will enable listening for those signals. Also accepts `ALL` or `NONE` to listen to all system signals, or ignore all system signals.
```

Comment:
Maybe list explicitly the signals that are handled by default...leaving a note in the impl to update that if more signals are added.

I would probably use 'all' and 'none' as the special-case words to distinguish more clearly from explicit signals.

Also specify here if this is case-sensitive.

Reply:
I think I want to keep SIGTERM and SIGUSR1 instead of TERM and USR1 for specificity.